scrapers_next/va/committees.py

```diff
@@ -0,0 +1,149 @@
+                role = "Member"
+
+            detail_link = member_items[i].get("href")
+            time.sleep(4)
+            # .do_scrape() allows us to get information from MemberDetail without
+            # returning/writing a com object to disk
+            com = [
+                i
+                for i in MemberDetail(
+                    [com, role], source=URL(detail_link, timeout=120)
+                ).do_scrape()
+            ][0]
+
+        return com
+
+
+class FindSubCommittees(HtmlListPage):
+
+    selector = CSS("#mainC > ul:nth-child(12) > li a")
+
+    def process_item(self, item):
+        try:
+            comm_name = item.text
+
+        except SelectorError:
+            raise SkipItem("no subcommittees")
+
+        chamber_text = item.getparent().getparent().getparent().getchildren()[1].text
+        if "house" in chamber_text.lower():
+            chamber = "lower"
+            parent_comm = chamber_text.split("House")[1].strip()
+        else:
+            chamber = "upper"
+            parent_comm = chamber_text.split("Senate")[1].strip()
+
+        com = ScrapeCommittee(
+            name=comm_name,
+            classification="subcommittee",
+            chamber=chamber,
+            parent=parent_comm,
+        )
+
+        detail_link = item.get("href")
+        com.add_source(self.source.url, note="homepage")
+        com.add_source(detail_link)
+
+        return CommitteeDetail(com, source=URL(detail_link, timeout=120))
+
+
+class CommitteeList(HtmlListPage):
+    source = "https://lis.virginia.gov/231/com/COM.HTM"
+    selector = CSS(".linkSect a")
+
+    def process_item(self, item):
+        comm_name = item.text
+        print(comm_name)
+        # both senate and house committees are listed on one page, so this isolates which is which
+        chamber_text = item.getparent().getparent().getparent().getchildren()[0].text
+        if "HOUSE" in chamber_text:
+            chamber = "lower"
+        else:
+            chamber = "upper"
+
+        com = ScrapeCommittee(
+            name=comm_name,
+            classification="committee",
+            chamber=chamber,
+        )
+
+        detail_link = item.get("href")
+        com.add_source(self.source.url, note="homepage")
+        com.add_source(detail_link)
+        time.sleep(3)
+
+        return CommitteeDetail(com, source=URL(detail_link, timeout=120))
+
+
+class SubCommitteeList(HtmlListPage):
+    source = "https://lis.virginia.gov/231/com/COM.HTM"
+    selector = CSS(".linkSect a")
+
+    def process_item(self, item):
+        chamber_text = item.getparent().getparent().getparent().getchildren()[0].text
+        comm_name = item.text
+
+        # this is hacky but I couldn't figure out another way to ignore a lack of subcommittees
+        if "SENATE" in chamber_text:
+            if comm_name == "Transportation":
+                raise SkipItem("no subcommittees")
+            if comm_name == "Rules":
+                time.sleep(4)
```

Comment:
Un-commented `sleep` statements are usually a bad sign. Why do we need to sleep for 4 seconds here?